website/settings/defaults.py

```diff
@@ -641,6 +641,12 @@ class CeleryConfig:
                 'task': 'management.commands.update_institution_project_counts',
                 'schedule': crontab(minute=0, hour=9), # Daily 05:00 a.m. EDT
             },
+            'archive_registrations_on_IA': {
```

Comment:
This needs to be added to the list of priority modules as well as imports

Reply:
This still needs to be in `imports`